test/language/statements/class/decorator/context-name.js

```diff
@@ -0,0 +1,31 @@
+/*---
+description: Decorator @ DecoratorCallExpression (Valid syntax for decorator on class.)
+esid: pending
+features: [class, decorators]
+includes: [compareArray.js]
+info: |
+  This was taken from the babel test suite https://github.com/babel/babel/blob/87ec4fc3d2cf44c30ef007f7da47d7177e07ffec/packages/babel-plugin-proposal-decorators/test/fixtures/2023-11-accessors/context-name/input.js#L1
+
+---*/
+
+const logs = [];
+const dec = (value, context) => { logs.push(context.name) };
+const f = () => { logs.push("computing f"); return { [Symbol.toPrimitive]: () => (logs.push("calling toPrimitive"), "f()") }; };
+class Foo {
+  @dec static accessor a;
+  @dec static accessor #a;
+
+  @dec static accessor "b"
+  @dec static accessor ["c"];
+
+  @dec static accessor 0;
+  @dec static accessor [1];
+
+  // TODO these two throw an error with SyntaxError: Unexpected token ILLEGAL but they pass on babel's test suite. should we remove them?
+  @dec static accessor 2n;
+  @dec static accessor [3n];
```

Comment:
These are currently failing when I use the babel-test262-runner to test them but are working upstream in babel so I don't know if I should keep these two cases 🤔 thoughts?